packages/common/src/search/Collection.ts

```diff
@@ -1,36 +1,68 @@
-import { hubSearchQuery, IArcGISContext } from "..";
+import { cloneObject, IArcGISContext } from "..";
 import {
   EntityType,
-  IHubCatalog,
   IHubCollection,
   IHubSearchOptions,
   IHubSearchResponse,
   IHubSearchResult,
   IQuery,
 } from "./types";
-
+import { hubSearch } from "./hubSearch";
 /**
- * Implements Collection behavior
+ * Collection Class
+ *
+ * Abstracts searching a Collection
+ *
+ * For more information, check out the [Catalog & Collection Guide](/hub.js/guides/concepts/catalog-collection/)
  */
-/* istanbul ignore next */
 export class Collection implements IHubCollection {
   private _context: IArcGISContext;
   private _collection: IHubCollection;
-  private constructor(collection: IHubCollection, context?: IArcGISContext) {
+  private constructor(collection: IHubCollection, context: IArcGISContext) {
     this._collection = collection;
-    if (context) {
-      this._context = context;
-    }
+    this._context = context;
   }
 
-  public static async create(
+  /**
+   * TBD if we implement this
+   * Fetch a collection from a catalog stored with an item
+   * @param collection
+   * @param context
+   * @returns
+   */
+  // public static async init(
```

Comment:
It might just be simpler to simply fetch the catalog of the resource (as we do in Catalog) and then filter by identifier.

Reply:
Agreed - the tricky bit is more in how to specify the collection key, for Enterprise Site urls, which already have a hash in them. I tried to explore this with some of the Enterprise test servers, but apparently they all have CORS locked down tight so I can't really test permutations. Until I find a server w open CORS, I figured I'd just comment this out. 

Also - since most collections are actually generated dynamically and passed into galleries, it's not super high-value at this point